apps/website/cli/updatePbSettings.ts

```diff
@@ -17,7 +17,7 @@ export default {
         const websiteOutput = getStackOutput({ folder: "apps/website", env: args.env });
 
         // 2. Get exports from `api` stack, again, for `args.env` environment.
-        const apiOutput = getStackOutput({ folder: "api-ddb", env: args.env });
+        const apiOutput = getStackOutput({ folder: "api", env: args.env });
```

Comment:
There was an `if` here in my last review. Let's bring it back as the warning for the user will be informative.